clientv3/auth.go

```diff
@@ -41,8 +42,11 @@ type Auth interface {
 	// UserChangePassword changes a password of a user.
 	UserChangePassword(ctx context.Context, name string, password string) (*AuthUserChangePasswordResponse, error)
 
-	// RoleAdd adds a new user to an etcd cluster.
+	// RoleAdd adds a new role to an etcd cluster.
 	RoleAdd(ctx context.Context, name string) (*AuthRoleAddResponse, error)
+
+	// RoleGrant grants a role to a user.
+	RoleGrant(ctx context.Context, name string, readPerm bool, writePerm bool, prefix bool, path string) (*AuthRoleGrantResponse, error)
```

Comment:
I feel RoleGrant should be granting perms to roles. UserGrant should be granting roles to users. The current RoleGrant does too much?